bound/kt/pom.xml

```diff
@@ -102,6 +102,7 @@
         <version.com.fasterxml.jackson>2.17.0</version.com.fasterxml.jackson>
         <version.net.java.dev.jna>5.12.0</version.net.java.dev.jna>
         <version.org.jetbrains.kotlinx.kotlinx.coroutines.core>1.8.1</version.org.jetbrains.kotlinx.kotlinx.coroutines.core>
+        <version.xyz.block.web5>commit-f28d90c-SNAPSHOT</version.xyz.block.web5>
```

Comment:
this is currently using a snapshot build, but that's because we don't have an official web5-kt release yet -- will follow up with the official release once its available